temporalcli/commandsgen/commands.yml

```
@@ -394,6 +394,160 @@ commands:
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Deployment commands perform operations on Worker Deployments:
+
+      ```
+      temporal deployment [command] [options]
+      ```
+
+      For example:
+
+      ```
+      temporal deployment list
+      ```
+    option-sets:
+      - client
+    docs:
+      description-header: >-
+        Temporal Deployment commands enable operations on Worker Deployments,
+        such as describe, list, update-current, and get-current, simplifying
+        versioning and management of workers.
+      keywords:
+        - deployment
+        - deployment describe
+        - deployment list
+        - deployment get-current
+        - deployment update-current
+
+  - name: temporal deployment describe
+    summary: Show properties of a Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Describes properties of a Deployment, such as whether it is current,
+      some custom metadata, or a list of its task queues.
+
+      ```
+      temporal deployment describe [options]
+      ```
+
+      For example, to also include reachability information:
+
+      ```
+      temporal deployment describe \
+          --deployment-series-name YourDeploymentSeriesName \
+          --deployment-build-id YourDeploymentBuildId \
+          --report-reachability
+      ```
+    option-sets:
+      - deployment-reference
+    options:
+      - name: report-reachability
+        type: bool
+        description: |
+          Flag to include reachability information of a Deployment.
+
+  - name: temporal deployment get-current
+    summary: Show the current Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Gets the current Deployment for a Deployment Series Name. Whether a
+      Deployment is current or not can affect which Workers will execute
+      Tasks of an existing or new Workflow.
+
+
+      ```
+      temporal deployment get-current [options]
+      ```
+
+      For example:
+
+      ```
+      temporal deployment get-current \
+          --deployment-series-name YourDeploymentSeriesName
+      ```
+    options:
+      - name: deployment-series-name
```

```suggestion
      - name: series-name
```

Do you need to prefix many of these options with `deployment-`?

---

Maybe not, I'm on the fence... It is a bit verbose, but `series-name` is always part of a deployment, but the previous command scopes for `deployment` anyway...

---

I'm with chad.

---

OK,  removing `deployment-`...